src/test/catch.cpp

```diff
@@ -0,0 +1,9430 @@
+//              Copyright Catch2 Authors
```

Comment:
Is it possible to reuse the `Catch2` version in core: https://github.com/realm/realm-core/tree/master/external? If no, I would prefer to include it as a git submodule.

Reply:
It's possible, but I didn't have time to read the the extensive documentation in realm-core explaining the way they use it. 

Also pressed by time have to go with something simple and that solve our problem.